cli/required.go

```diff
@@ -14,15 +12,20 @@ func NoArgs(cmd *cobra.Command, args []string) error {
 	}
 
 	if cmd.HasSubCommands() {
-		return errors.Errorf("\n" + strings.TrimRight(cmd.UsageString(), "\n"))
+		return errors.Errorf(
+			"%[1]s: unknown command: %[2]s %[3]s\n\nUsage:  %[4]s\n\nRun '%[2]s --help' for more information",
```

Comment:
For consistency with other error messages where we print out commands the user has inputted (or could input, like `Run '%[2]s --help' for more information` further in this message), maybe it could make sense to surround `%[2]s %[3]s` with  single quotes`'` as well.  

I think it could make the message a little bit more readable, given we are prefixing `docker:` as well. WDYT?

Reply:
OHMAN I went back and forth on some of these; initially I had the quotes, but then also looked at `unknown flag:` which didn't have them; that one is generated by Cobra  (or spf13/flags), so not something I could immediately change.

Overall I somewhat think that the quotes may be good to add yes; perhaps still ok for a follow-up pass?

Reply:
Yeah for sure! It's definitely a minor thing, but if users are anything like me they do appreciate consistency in the outputs. We'll get to it in a separate pass 👍